### ISSUE TITLE
Inconsistent highlighting for incrementing and decrementing variables

### ISSUE THREAD

**Issue by RCRalph** [worlpaker/go-syntax#2]
Hi, I found an inconsistency in syntax highlighting when incrementing and decrementing variables in loops.  If in the third part of a for-loop we decrement a variable, the `--` signs are blue instead of white. Below you can find an example of this behavior:

![image](https://user-images.githubusercontent.com/36704328/229633436-2dad5952-4c66-4b9c-a409-39961197f554.png)

```go
package main

func main() {
    for i := 0; i < 10; i++ {
        // Stuff
    }

    for i := 0; i < 10; i-- {
        // Stuff
    }

    i++
    i--
}
```

I'm open to working with you to fix this issue, let me know if there's anything I can help you with.

**Comment by worlpaker** [comment]
Hi,

Thank you for your time and contribution. 

Yes, you are right. Decrementing operator in the for loop should be white.

**v0.2.6 - Dark+ Theme**
![fixed_#2](https://user-images.githubusercontent.com/34960716/229648720-6f007ecd-fefc-4f99-8fd7-cb8f718186af.png)



This bug has been fixed in v0.2.6 thanks to you.

**Comment by RCRalph** [comment]
Thank you for the quick fix
